Decred Wallet/Features/Navigation Menu/NavigationMenuViewController.swift

```diff
@@ -183,52 +183,51 @@ extension NavigationMenuViewController: SyncProgressListenerProtocol {
         }
     }
     
     func onHeadersFetchProgress(_ progressReport: DcrlibwalletHeadersFetchProgressReport) {
         self.handleGeneralProgressReport(progressReport.generalSyncProgress!)
         
-        self.syncStatusLabel.text = "Fetching block headers."
+        self.syncStatusLabel.text = LocalizedStrings.fetchingBlockHeaders
         if progressReport.currentHeaderTimestamp != 0 {
-            self.bestBlockLabel.text = "\(progressReport.totalHeadersToFetch - progressReport.fetchedHeadersCount) blocks behind."
+            self.bestBlockLabel.text = String(format: LocalizedStrings.blocksBehind, progressReport.totalHeadersToFetch - progressReport.fetchedHeadersCount)
             if progressReport.bestBlockAge != "" {
-                self.bestBlockAgeLabel.text = "\(progressReport.bestBlockAge) ago"
+                self.bestBlockAgeLabel.text = String(format: LocalizedStrings.bestBlockAgeAgo, progressReport.bestBlockAge)
             }
         }
     }
     
     func onAddressDiscoveryProgress(_ progressReport: DcrlibwalletAddressDiscoveryProgressReport) {
         self.handleGeneralProgressReport(progressReport.generalSyncProgress!)
         
-        self.syncStatusLabel.text = "Discovering used addresses."
-        self.bestBlockLabel.text = "\(progressReport.generalSyncProgress!.totalSyncProgress)% completed, \(progressReport.generalSyncProgress!.totalTimeRemaining) left."
+        self.syncStatusLabel.text = LocalizedStrings.discoveringUsedAddresses
+        self.bestBlockLabel.text = String(format: LocalizedStrings.generalSyncProgressCompletedleft, progressReport.generalSyncProgress!.totalSyncProgress,progressReport.generalSyncProgress!.totalTimeRemaining)
         self.bestBlockAgeLabel.text = ""
     }
     
     func onHeadersRescanProgress(_ progressReport: DcrlibwalletHeadersRescanProgressReport) {
-        self.syncStatusLabel.text = "Scanning blocks."
+        self.syncStatusLabel.text = LocalizedStrings.scanningBlocks
         self.bestBlockAgeLabel.text = ""
         
         if progressReport.generalSyncProgress == nil {
             // generalSyncProgress is nil during rescan.
             self.refreshBestBlockAgeTimer?.invalidate()
-            self.bestBlockLabel.text = "\(progressReport.rescanProgress)% completed, \(progressReport.timeRemaining) left."
+            self.bestBlockLabel.text = String(format: LocalizedStrings.rescanProgress, progressReport.rescanProgress,progressReport.timeRemaining)
             return
         }
-        
         self.handleGeneralProgressReport(progressReport.generalSyncProgress!)
-        self.bestBlockLabel.text = "\(progressReport.generalSyncProgress!.totalSyncProgress)% completed, \(progressReport.generalSyncProgress!.totalTimeRemaining) left."
+        self.bestBlockLabel.text = String(format: LocalizedStrings.syncTotalProgress, progressReport.generalSyncProgress!.totalSyncProgress,progressReport.generalSyncProgress!.totalTimeRemaining)
```

Comment:
There's a comma in this line without a succeeding space (space after the comma). There should be a single space after each comma to separate the arguments in the function.